spec/rspec/core/lookup_set_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe 'RSpec::Core::LookupSet' do
+
+  let(:set) { RSpec::Core::LookupSet.new([1, 2, 3]) }
+
+  it 'takes an array of values' do
+    expect(set).to include(1).and include(2).and include(3)
+  end
+
+  it 'can be appended to' do
+    set << 4
+    expect(set).to include 4
+  end
+
+  it 'can have more values merged in' do
+    set.merge([4, 5]).merge([6])
+    expect(set).to include(4).and include(5).and include(6)
+  end
+
+  it 'is enumerable' do
+    expect(
+      set.inject([]) { |array, value| array << value }
+    ).to eq [1, 2, 3]
+  end
```

Comment:
This spec shows it has an `inject` method that behaves like enumerable's but it doesn't show that the set is actually enumerable.  What do you think of this instead?

``` ruby
expect(set).to be_an Enumerable
expect { |p| set.each(&p) }.to yield_successive_args(1, 2, 3)
```

That shows that the set is an `Enumerable` and that it implements the contract required by classes that mixin `Enumerable` (i.e. that they implement `each` properly).